src/components/Slider/services/sliderFetch.ts

```diff
@@ -0,0 +1,7 @@
+import axios, { AxiosResponse } from 'axios';
+import type { Slides } from '../types/SliderTypes';
+
+export const fetchData = async (url: string): Promise<Array<Slides>> => {
+  const response: AxiosResponse = await axios.get(url);
+  return response.data.results;
+};
```

Comment:
move to shared/api

Reply:
here is an approach, please discuss it with Volodymyr:
https://github.com/kerrimov/the-ssspace/pull/20/files#diff-ec8cd247324ad888f785b5ff8e963f25aebb4596cb88c30b60ef73eae04210e7

Reply:
moved to shared and added generic for whole team